code/obj/rocko.dm

```diff
@@ -0,0 +1,184 @@
+
+// CE's pet rock! A true hellburn companion
+obj/item/rocko
+	name = "Rocko"
+	icon = 'icons/obj/materials.dmi'
+	icon_state = "rock1"
+	w_class = 1
+	force = 10
+	throwforce = 15
+	throw_range = 3
+
+	var/static/list/rocko_is
+	var/smile = TRUE
+	var/painted
+	var/bright = FALSE
+	var/mob/living/holder
+	var/obj/item/clothing/head/hat
+
+	New()
+		. = ..()
+		var/matrix/xf = matrix()
```

Comment:
cross frequency?

Reply:
x factor?

Reply:
academic dishonesty?

Reply:
xenforo?

Reply:
extra fine?

Reply:
extensible format?

Reply:
**1**? I? id? unit matrix? Einheitsmatrix?

Reply:
Hmmm I guess it should be: A

Reply:
Removed variable as the identity matrix can part of the assignment.